github/actions_runners_test.go

```diff
@@ -225,6 +225,56 @@ func TestActionsService_ListOrganizationRunners(t *testing.T) {
 	}
 }
 
+func TestActionsService_ListEnabledReposInOrg(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/orgs/o/actions/permissions/repositories", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testFormValues(t, r, values{
+			"page": "1",
+		})
+		fmt.Fprint(w, `{"total_count":2,"repositories":[{"id":2}, {"id": 3}]}`)
+	})
+
+	ctx := context.Background()
+	opt := &ListOptions{
+		Page: 1,
+	}
+	got, _, err := client.Actions.ListEnabledReposInOrg(ctx, "o", opt)
+	if err != nil {
+		t.Errorf("Actions.ListEnabledReposInOrg returned error: %v", err)
+	}
+
+	want := &ActionsEnabledOnOrgRepos{TotalCount: int(2), Repositories: []*Repository{
+		{ID: Int64(2)},
+		{ID: Int64(3)},
+	}}
+	if !reflect.DeepEqual(got, want) {
+		t.Errorf("Actions.ListEnabledReposInOrg returned %+v, want %+v", got, want)
+	}
+
+	// Test addOptions failure
```

Comment:
Is this testing an invalid owner, or bad list options? It looks like opt hasn't changed since the previous (valid) call that was tested. Is the error returned actually related to the arguments passed to the function, or is it an error about not finding the endpoint because the test only has an endpoint for `/orgs/o/actions/permissions/repositories` registered?

Reply:
Hey @nightlark :)! I am not sure if I understand your comment, could you give me some pointers?

Reply:
The comment about it testing addOptions doesn't quite feel like it fits what is happening (a newline in the middle of a url where the owner belongs).

I found ~4 other tests with the same thing so I guess that the tests you added were based on one of those. Not sure the comment is accurate in any of those cases, or is really needed in any of those cases (I think throwing a newline in the middle of a url by using \n as the owner or organization argument to any go-github function is likely to give an error, so in a way the test seems kinda pointless -- gmlewis maybe you have some insight I'm missing).

Given that the same sort of test case happens in a few other places, I don't think leaving another instance of it in this test does any harm.

Reply:
Hi @nightlark - yes, this has been added simply to increase the code coverage reports.  Adding in the newline is basically an easy way to generate an error, so it exercises the error-handling path.